dpypelines/pipeline/configuration.py

```diff
@@ -23,13 +25,22 @@
     # This *always* needs to be the final entry in the dictionary to prevent inadvertent matches
     "^.*$": {
         "config_version": 1,
-        "transform": sdmx_compact_2_0_prototype_1,
+        "transform": sdmx_generic_2_1_prototype_1,
         "transform_inputs": {"^data.xml$": sdmx_sanity_check_v1},
         "transform_kwargs": {},
         "required_files": [{"matches": "^data.xml$", "count": "1"}],
         "supplementary_distributions": [{"matches": "^data.xml$", "count": "1"}],
         "secondary_function": dataset_ingress_v1,
     },
+    "*._move$": {
+        "config_version": 1,
+        "transform": None,
+        "transform_inputs": {},
+        "transform_kwargs": {},
+        "required_files": [{"matches": "^data.xml$", "count": "1"}],
+        "supplementary_distributions": [{"matches": "^data.xml$", "count": "1"}],
```

Comment:
supplementary_distributions is for files to upload alongside outpuuts from transforms. There's no transform, this can be blank.

Reply:
done